bin/build-constraints.sh

```diff
@@ -8,13 +8,13 @@ run_from_hook=${1:-''}
 
 readonly policies_file="$ENV_DIR/env/policies.yaml"
 readonly output_path="/tmp/otomi/constraints"
-readonly crd_artifacts_path="$PWD/charts/gatekeeper-artifacts/crds"
 
 # build yaml resources from policy files
 function build() {
   echo "Building constraints artifacts from policies."
   local policies_path="./policies"
-  rm -f $output_path/* $crd_artifacts_path/template_*
+  mkdir -p $output_path
+  rm -f $output_path/*
```

Comment:
why create a new empty folder and try to remove something in there in the next line

Reply:
T he folder is not present in the first place. Secondly, it needs to be cleared in case there were files from previous run.

Reply:
Also, I am still pushing some changes, not ready to merge yet!

Reply:
The mkdir will fail if it exists, so that won't work.

Reply:
are you sure? maybe it works!